scss/components/_citationDialog.scss

```diff
@@ -632,7 +632,9 @@
 						padding-inline-start: 100px;
 					}
 					#label {
-						flex: 1;
+						// prevent left-edge of <select> from expanding past the "Remove" button
+						// for locales that have long labels
+						max-width: 138px;
```

Comment:
What is 138px?

Did you check whether this is enough space for all locales not to be truncated?